src/components/MemberDetails.tsx

```diff
@@ -1,17 +1,29 @@
-import { Member } from "pages";
 import { Serif, Link } from "@artsy/palette";
 import { formatDistanceToNow } from "date-fns";
 import { capitalize } from "lodash-es";
 import RouterLink from "next/link";
-import { normalizeParam } from "utils";
 import { Fragment } from "react";
 import { Grid } from "components/Grid";
+import { Location, Member, Organization, Subteam, Team } from "@prisma/client";
 
-interface MemberDetailsProps {
-  member: Member;
+interface MemberDetails {
+  member: Member & {
+    orgs: Organization[];
+    locations: Location[];
+    teams: Team[];
+    subteams: Subteam[];
+    manager: {
+      name: string;
+      slug: string;
+    };
+    reports: {
+      name: string;
+      slug: string;
+    }[];
+  };
 }
```

Comment:
Sometimes it was hard to extract the types of what a query result would be to pass it around to different places. It'd be easier if I centralized all the queries, which I may do... but for now there's a few places w/ this pretty verbose/ugly type redeclaration.